scholarx/archive/index.html

```diff
@@ -195,6 +212,32 @@ <h6>UNIVERSITY</h6>
     {{/mentorProfiles}}     
 </script>
 
+<!-- template mentees -->
+<script type="text/html" id="templateMentees">
+    {{#menteeProfiles}}
+        <tr >
+            <td>{{name}}</td>
+            <td>{{mentor}}</td>
+        </tr>
+    {{/menteeProfiles}}     
+</script>
+
+<!--mentor mentee transition-->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>      
+<script>
+$(document).ready(function(){
+    $('#myselection').on('change', function(){
+    	var demovalue = $(this).val();
```

Comment:
demovalue -> selectedValue

Reply:
Anjula Can you check now?
I didn't show the profile photo and uni details because we don't have those details for some of our mentees.
So I commented that part in the code.

Reply:
@janithlahirukariyawasam let's uncomment them, we can keep it there. Since we have not published this page we can keep it because in the future we'll add the data.

Reply:
@anjula-sack Uncommented uni and profile photo details.